### DISCUSSION TITLE
Sort frames in the training set by # of nodes 'approved' (teal labels) per frame

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1386] [Ideas]
When labelling frames by correcting predicted instances from an existing model, sometimes I'll accidentally miss a node on a frame (node is still red and is in the training set, but incorrectly labelled), and then that mistake gets folded into the training data.

It would be great if there was a way to easily sort frames in the project by %nodes/instance that have been turned teal.

A quick way to do this would be to add a hotkey to cycle through frames that still have gray (unlabelled) and red (unapproved but included in the training set) nodes.

A more user-friendly way would be to add a fifth GUI tab (in addition to Skeleton/Instances/Videos/Labeling Suggestions) that lists all frames in the training set, where there is a column for %nodes/instance that are teal. That way I could sort all frames, and quickly check that the frames that don't have a score of 100% make sense (nodes are outside the field of view, etc).

Thanks!
![SLEAPdemo](https://github.com/talmolab/sleap/assets/69451171/586185c5-2d74-45a7-9330-4e72df8b8e85)

**Comment by talmo** [comment]
Great idea!
